nanoFirmwareFlasher.Tool/Options.cs

```diff
@@ -341,17 +339,51 @@ public class Options
             HelpText = "Reads details from connected device.")]
         public bool DeviceDetails { get; set; }
 
+        [Option(
+            "showfwonly",
```

Comment:
Shouldn't this be "identify firmware" instead? After all the end goal is to _identify_ the firmware package to use. _Showing_ it is there to... show it... ;)

Reply:
That is a better name. I'll turn it into identifyfwversion on the next commit.